tests/test_run_experiment.py

```diff
@@ -26,12 +26,6 @@ def test_run_experiment():
     config['test_domains'] = []
     run_experiment(config, param_file, param_index, out_file)
 
-
-def test_apply_dict_to_args():
```

Comment:
we don't need this test anymore? I would say we do not have to delete test as coverage does not say everything

Reply:
ah, ok, moved to test_argparse.py